controllers/etcd_controller.go

```diff
@@ -254,6 +258,20 @@ func (r *EtcdReconciler) reconcile(ctx context.Context, etcd *druidv1alpha1.Etcd
 		}, err
 	}
 
+	if etcd.Spec.Backup.CopySpec != nil {
+		values, err := r.getMapFromEtcd(etcd)
+		if err != nil {
+			return ctrl.Result{Requeue: true}, err
```

Comment:
Let's match the style of other returns
```suggestion
			return ctrl.Result{
				Requeue: true,
			}, err
```